Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8 
+
+WORKDIR /usr/src/GENIE-Sponsored-Projects
+
+COPY requirements.txt ./ 
+RUN pip install --no-cache-dir -r requirements.txt 
```

Comment:
If you copy the files already, you can do a pip install after I think.  Also this package is installable by pip, so I think you could do something like `pip install . ` and the CMD would be `geniesp ...`

Reply:
rearranged Dockerfile to run pip installation of requirements after consolidated copying operation.  if installing geniesp from pip, this would only be a stable, fixed version?  considering how many adaptations needed, it's nice to be able to dockerize code from a dev branch